renovate.json

```diff
@@ -107,11 +107,21 @@
         "task/validate-fbc/**"
       ]
     },
+    {
+      "groupName": "preflight-deprecated",
+      "description": "contains the group of preflight tasks that no longer work in the konflux ecosystem with the latest preflight container image",
+      "matchFileNames": [
+        "task/ecosystem-cert-preflight-checks/01/**"
+      ],
+      "enabled": false
+    },
     {
       "groupName": "preflight",
+      "description": "contains the group of current active preflight tasks",
       "matchFileNames": [
         "task/ecosystem-cert-preflight-checks/**"
-      ]
+      ],
+      "enabled": true
```

Comment:
It seems that this group has to be first in the list, because:

* `task/ecosystem-cert-preflight-checks/**` matches everything, including `task/ecosystem-cert-preflight-checks/0.1`
* attributes from later groups in the list override those from earlier groups, so `enabled` flips back to true.

Swapping the order of the `preflight` and `preflight-deprecated` groups should fix it

Reply:
I swapped the order of these and made your other update as well. Thanks for looking at this and testing.